docusaurus.config.js

```diff
@@ -22,10 +22,10 @@ module.exports = {
   ],
   themeConfig: {
     announcementBar: {
-      id: 'shimmer',
+      id: 'assembly',
       content:
-        'Interested in the upcoming incentivized staging network Shimmer, and in staking your IOTA tokens? Read more about it <a target="_blank" href="https://shimmer.network">here</a>.',
-      backgroundColor: '#00e0ca',
+        'Interested in the upcoming permissionless Smart Contracts network Assembly, and in staking your IOTA tokens? Read more about it <a target="_blank" href="https://assembly.sc">here</a>.',
```

Comment:
```suggestion
        'Would you like to stake your tokens? <a target="_blank" href="https://assembly.sc">Read more</a> about Assembly, the upcoming permissionless Smart Contract network.',
```

I used the full sentence structure, moved the heaviest part to the end and have shortened the sentence a bit.

Reply:
❤️